src/test/resources/application-test.yml

```diff
@@ -32,6 +32,9 @@ complexity_of_need_uri: http://localhost:10000
 spring:
   datasource:
     url: "jdbc:hsqldb:mem:keyworker-db;sql.syntax_pgs=true;shutdown=false"
+  flyway:
```

Comment:
Same as https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/330#discussion_r1162577187

Reply:
No need to resolve, done already